src/Microsoft.SqlTools.ResourceProvider.Core/Contracts/Account.cs

```diff
@@ -58,7 +58,55 @@ public IEnumerable<Tenant> Tenants
             get;
             set;
         }
+        /// <summary>
+        /// Information about the auth provider
+        /// </summary>
+        public ProviderSettings ProviderSettings;
+
+    }
+
+    public class ProviderSettings
+    {
+        /// <summary>
+        /// Display name of the provider 
+        /// </summary>
+        public string DisplayName;
+
+        /// <summary>
+        /// ID of the provider
+        /// </summary>    
+        public string Id;
+        /// <summary>
+        /// Settings for the provider itself
+        /// </summary>
+        public ProviderSettingsObject Settings;
```

Comment:
Is there a reason this is in its own object? Seems odd to have a settings class contain its own settings var

Reply:
It's just to mimic the object we get from ADS. I didn't want to change it __too much__.